docs/guide/content/user_guide/uploading_files.md

```diff
@@ -29,13 +29,13 @@ Uploads are organized into **Upload Bundles**, which group a set of files and li
 
    Once these steps are complete, the upload bundle is linked to the appropriate dataset and ready for file staging.
 
-4. **Add Files**  
+4\. **Add Files**  
```

Comment:
This shows up as a 1. We should make it a 4 somehow, maybe with this hack?!?

Reply:
The above format fixes this and can be removed

Reply:
fixed without doing the workaround